File: docs/components_page/components/breadcrumb/active.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

breadcrumb <- htmlDiv(
    list(
        dbcBreadcrumb(
            list(
                dbcBreadcrumbItem("Docs", href="/docs/", active="exact"),
                dbcBreadcrumbItem(
                    "Components", href="/docs/components/", active="exact"
                ),
                dbcBreadcrumbItem(
                    "Breadcrumb",
                    href="/docs/components/breadcrumb/",
                    active="exact",
                )
            )
        ),
        dbcBreadcrumb(
            list(
                dbcBreadcrumbItem("Docs", href="/docs/", active="partial"),
                dbcBreadcrumbItem(
                    "Components", href="/docs/components/", active="partial"
                ),
                dbcBreadcrumbItem(
                    "Breadcrumb",
                    href="/docs/components/breadcrumb/",
                    active="partial",
                )
            )
        ),
        dbcBreadcrumb(
            list(
                dbcBreadcrumbItem("Docs", href="/docs/"),
                dbcBreadcrumbItem(
                    "Components", href="/docs/components/", active=TRUE
                ),
                dbcBreadcrumbItem(
                    "Breadcrumb", href="/docs/components/breadcrumb/"
                )
            )
        )
    )
)

File: docs/components_page/components/breadcrumb/active.jl
using DashBootstrapComponents, DashHtmlComponents

breadcrumb = html_div(
    [
        dbc_breadcrumb(
            [
                dbc_breadcrumbitem("Docs", href="/docs/", active="exact"),
                dbc_breadcrumbitem(
                    "Components", href="/docs/components/", active="exact"
                ),
                dbc_breadcrumbitem(
                    "Breadcrumb",
                    href="/docs/components/breadcrumb/",
                    active="exact",
                ),
            ],
        ),
        dbc_breadcrumb(
            [
                dbc_breadcrumbitem("Docs", href="/docs/", active="partial"),
                dbc_breadcrumbitem(
                    "Components", href="/docs/components/", active="partial"
                ),
                dbc_breadcrumbitem(
                    "Breadcrumb",
                    href="/docs/components/breadcrumb/",
                    active="partial",
                ),
            ],
        ),
        dbc_breadcrumb(
            [
                dbc_breadcrumbitem("Docs", href="/docs/"),
                dbc_breadcrumbitem(
                    "Components", href="/docs/components/", active=true
                ),
                dbc_breadcrumbitem(
                    "Breadcrumb", href="/docs/components/breadcrumb/"
                ),
            ],
        ),
    ]
)
